src/plugins/condition/components/Criterion.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+<li class="has-local-controls t-condition">
+    <label>{{ setRowLabel }}</label>
+    <span class="t-configuration">
+        <span class="controls">
+            <select v-model="selectedTelemetryObject"
+                    @change="updateFieldOptions"
+            >
+                <option value="">- Select Telemetry -</option>
+                <option v-for="telemetryOption in telemetry"
+                        :key="telemetryOption.identifier.key"
+                        :value="telemetryOption.identifier"
+                >
+                    {{ telemetryOption.name }}
+                </option>
+            </select>
+        </span>
+        <span class="controls">
+            <select v-model="selectedFieldOption"
+                    @change="persist"
+            >
+                <option value="">- Select Field -</option>
+                <option v-for="option in telemetryMetadata"
+                        :key="option.key"
+                        :value="option"
+                >
+                    {{ option.name }}
+                </option>
+            </select>
+        </span>
+        <span class="controls">
+            <select v-model="selectedOperationOption"
+                    @change="updateOperationInput"
+            >
+                <option value="">- Select Comparison -</option>
+                <option v-for="option in operations"
+                        :key="option.name"
+                        :value="option.name"
+                >
+                    {{ option.text }}
+                </option>
+            </select>
+            <input v-if="isInputOperation"
+                   v-model="comparisonInputValue"
+                   class="t-condition-name-input"
+                   type="text"
+                   @change="persist"
+            >
+        </span>
+    </span>
+</li>
+
+</template>
+
+<script>
+import { OPERATIONS } from '../utils/operations';
+
+export default {
+    inject: ['openmct'],
+    props: {
+        criterion: {
+            type: Object,
+            required: true
+        },
+        telemetry: {
+            type: Array,
+            required: true,
+            default: () => []
+        },
+        index: {
+            type: Number,
+            required: true
+        },
+        condition: {
```

Comment:
Doesn't look like `condition` prop is being used here. Please remove it.

Reply:
done